src/main/scala/dsptools/tester/VerilogTbDump.scala

```diff
@@ -25,7 +25,8 @@ trait VerilogTbDump {
 
   val (inputs, outputs) = TestersCompatibility.getDataNames("io", dut.io) partition {
     case (dat, name) =>
-      DataMirror.directionOf(dat) == chisel3.core.ActualDirection.Input
+      dat.dir == core.Direction.Input
+      //DataMirror.directionOf(dat) == chisel3.core.ActualDirection.Input
```

Comment:
What's going on here- is `DataMirror` changing again?

Reply:
No it didn't change, but the version of Chisel with ranges doesn't have DataMirror stuff, so I had to revert back until ranges get merged into Chisel master :(...